entrypoint.sh

```diff
@@ -10,6 +10,8 @@
 # Error codes
 E_ILLEGAL_ARGS=126
 
+SENTINEL_OVERLAY_CFG=/opt/sentinel-overlay
```

Comment:
Can we just call this `SENTINEL_OVERLAY`? The `CFG` part is a bit misleading since the overlay can also contain binary artifacts.